src/components/Tooltip.tsx

```diff
@@ -67,7 +63,12 @@ function Popper({ triggerRef, content, placement = "auto" }: PopperProps) {
   });
 
   return (
-    <div ref={popperRef} style={styles.popper} {...attributes.popper} css={Css.bgGray900.white.px1.py("4px").br4.xs.$}>
+    <div
+      ref={popperRef}
+      style={{ ...styles.popper, maxWidth: "320px" }}
```

Comment:
Adding a max-width here as the tooltip could get pretty long if the `content` didn't explicitly include line breaks. Waiting to confirm with design on what the right size is here.

Reply:
Confirmed with design. Will be going with 320px.